api/v2/utilities/validators.py

```diff
@@ -0,0 +1,44 @@
+import re
+from flask import request, jsonify, make_response
+from functools import wraps
+
+
+def validate_bouquet_adding(func):
+    """Decorator to validate the addition of a bouquet"""
+
+    @wraps(func)
+    def wrapper():
+        if func.__name__ == 'post':
+            request_data = request.get_json()
+
+            def check_numeric(_value):
+                _re = bool(re.fullmatch(r'^[a-zA-Z]*\s?[a-zA-Z]*\s?[a-zA-Z]*', str(value)))
```

Comment:
Python has a built-in `isnumeric` function that can save you the regular expression.